v3/plugins/scalameta/metals/1.9.6/meta.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v2
+publisher: scalameta
+name: metals
+version: 1.9.6
+type: VS Code extension
+displayName: Scala (Metals)
+title: Scala language server with rich IDE features
+description: This plug-in provides support for Scala development via the Metals language server.
+icon: https://raw.githubusercontent.com/scalameta/metals-vscode/v1.9.6/images/logo.png
+repository: https://github.com/scalameta/metals-vscode
+category: Language
+firstPublicationDate: "2020-11-10"
+spec:
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-scala:0.9.0-b92a5f9"
```

Comment:
quay.io/eclipse/che-plugin-sidecar:scala-0275d91
that's the latest (simplified) one you committed. 

then the volumes below should be gone to keep the ones coming from the sidecar